google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -115,3 +115,30 @@
 """
 
 # TODO: Add a RAW_GROUP_MEMBERS_TABLE.
+
+CREATE_BUCKETS_TABLE = """
+    CREATE TABLE `{0}` (
```

Comment:
Yep, those are to be added in the future and I'm preparing a separate PR for those and there should be a PR for that this week.